atspi-connection/p2p.md

```diff
@@ -0,0 +1,43 @@
+# P2P introduction
+
+Peer-to-Peer (P2P) support in [atspi](../atspi) enables direct connections to applications, bypassing the central accessibility bus when possible. This can significantly reduce method-call latency and improve performance, especially on systems with many accessible applications.
+
+## `p2p_tree` example
+
+```shell
+cargo run --release --example p2p_tree
+```
+
+The example walks all nodes in the accessibility tree and queries each node for its role. The example registers the amount of time the operation took.
+
+It does so in three ways:
+The `p2p_tree` example demonstrates three ways to traverse the accessibility tree and measure performance:
+
+1. **Bus:** Queries all nodes of all applications over the accessibility bus.
+2. **P2P Sequential:** Queries each application's nodes over a P2P connection (if available), one after another.
+3. **P2P Parallel:** Queries each application's nodes over P2P connections (if available) concurrently, leveraging parallelism.
```

Comment:
Parallelism and concurrency are different. I'd recommend not interweaving the two terms casually.

Parallelism refers to doing things _at the same time_ (i.e., using multi-processing), concurrency is about doing things _while waiting for other things to finish_ (i.e., check user interactions while waiting for an I/O stream to finish reading).

Even if your example used parallel processing (it doesn't), the application on the other side likely does not. The suffix `_concurrent` or similar fits better than a Rayon-like `_par`.